app/controllers/application_controller.rb

```diff
@@ -384,14 +384,14 @@ def viewers_hover(data, link)
     viewers = ''
 
     if link
-      viewers += t('shared.viewers_hover.visible_to')
+      viewers += t('shared.viewers.visible_to')
```

Comment:
ahh I thought I had caught all of these, but we need to eliminate this kind of string concatenation of partial English phrases... (this one builds up `Visible only ` + `to you`). I'm not sure what brakeman is complaining about but refactoring this to not concatenate strings would probably help.